layouts/default.vue

```diff
@@ -461,7 +460,12 @@ form {
 
 @media only screen and (min-width: 768px) {
   form {
+    min-width: 500px;
```

Comment:
Would a percentage value be better here like 80%. I don't normally like to use fixed widths on these types of things

Reply:
We're already using a % value on the line below (50%), there's just some edge case where after a certain point the container holding the input field and button gets smaller than the input field and button themselves causing the elements to not be centered at certain browser widths which is why I've added this fixed min-width

Reply:
Can you show me what it looks like without this min width value? At the minute I am wondering why this approach is better than a centering approach like flex?

Reply:
<img width="923" alt="Screenshot 2023-04-04 at 16 43 38" src="https://user-images.githubusercontent.com/129167065/229846066-8e9a92a8-3eef-485e-baa2-4a08b37b6ce3.png">